src/sync/impl/sync_file.hpp

```diff
@@ -104,10 +110,11 @@ class SyncFileManager {
 
 private:
     const std::string m_base_path;
+    const std::string m_app_id; // App ID's are guaranteed by MongoDB Realm to be lowercase and only consisting of [a-z0-9\-].
 
-    static constexpr const char c_sync_directory[] = "realm-object-server";
-    static constexpr const char c_utility_directory[] = "io.realm.object-server-utility";
-    static constexpr const char c_recovery_directory[] = "io.realm.object-server-recovered-realms";
+    static constexpr const char c_sync_directory[] = "mongodb-realm";
+    static constexpr const char c_utility_directory[] = "realm.mongodb.server-utility";
```

Comment:
These directories already reside in `mongodb-realm` - why do we need to redundantly repeat it at every level?

Reply:
Good point 👍